### PR TITLE
[IMP]migrate baidumap form v8 to V9.

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

File: website_baidu_map/website_baidu_map.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        <template id="event_description_full_inherited" name="map" inherit_id="website_event.event_description_full">
+            <xpath expr="//div[@class='panel-body']/a" position="replace">
+                <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+                <meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
+                <style type="text/css">
+                    body, html {width: 100%;height: 100%;margin:0;font-family:"微软雅黑";}
+                    #allmap{width:100%;height:250px;}
+                    p{margin-left:5px; font-size:14px;}
+                </style>
+                <script type="text/javascript" src="http://api.map.baidu.com/api?v=2.0&amp;ak=zFXL4R2eaZjkUSxsGDCAGLN8"></script>
```
Comment (elicoidal) [inline website_baidu_map/website_baidu_map.xml]: same remark than for v8: is it possible to have a property?

File: website_baidu_map/README.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+website_baidu_map
```
Comment (elicoidal) [inline website_baidu_map/README.rst]: Use proper English.

File: website_baidu_map/README.rst [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+=================
+
+This module extends the functionality of website to support baidu map in the contact us page.
```
Comment (elicoidal) [inline website_baidu_map/README.rst]: Maybe my code reading is improper but AFAIR, the map is link to event, not "contact us" page.
Can you recheck?

**Comment by Rona111** [comment]
cc @elicoidal @siyuancn @duanyp1991

File: website_event_baidu_map/README.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+ * have basic modules installed (website_event)
+
+
+Configuration
+
+=============
+
+To configure this module, you need to:
+
+* Go to Settings/Parameters/System Parameters and create a record which Key must be named baidu.api and the Value like [http://api.map.baidu.com/api]?v=[2.0]&ak=[your ak].
```
Comment (siyuancn) [inline website_event_baidu_map/README.rst:26]: Please add the description for how to get the application key for baidu map and test the module in https environment.

**Comment by victor-ditay** [comment]
@elicoidal @seb-elico @noahzaozao This PR is possible to be optimistic merged.

**Comment by victor-ditay** [comment]
> @elicoidal @seb-elico @noahzaozao This PR is possible to be optimistic merged.

I haven't seen there is a pending PR #87 for V8. 
This PR should wait to be merged the other. The migration should be regards first on the original latest version.

**Comment by seb-elico** [comment]
@victormartinelicocorp Agreed, let's wait for #87 to be merged. Right now, we focus on merging the v8 modules. Plus for v9, we need to add UT

**Comment by elicoidal** [comment]
@victormartinelicocorp What is the status here: this module would better fit in l10n-china IMO

**Comment by victor-ditay** [comment]
@elicoidal  This module as you see don't have any conflicts to be merged. Just worries me Travis but I think is a base problem on the master branch. And for you question I agree we should propose to OCA/l10n-china

1 Is blocking here Travis?
2 Should I propose now this module or wait to be merge this PR?

**Comment by elicoidal** [comment]
propose the module to OCA @victormartinelicocorp

**Comment by elicoidal** [comment]
propose the module to OCA @victormartinelicocorp

**Comment by victor-ditay** [comment]
> propose the module to OCA @victormartinelicocorp

Done in https://github.com/OCA/l10n-china/pull/31

cc @elicoidal @seb-elico @xuan139 @noahzaozao

**Pull request closed without merging**